rakelib/github_stats.rake

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-STATSD_ERROR_KEY = 'worker.github_stats_scraper.error'
+STATSD_METRIC = 'tasks.github_stats_scraper'
```

Comment:
J/w - is there anything else under `tasks.`?

Reply:
Nope, I opted to change it to `tasks` as it's not quite a worker and we don't have another similar bucket. 

Didn't spend a *ton* of time thinking about it but happy to change if there's a better idea